src/client.js

```diff
@@ -871,6 +871,18 @@ client.prototype.handleMessage = function handleMessage(message) {
               { plan, planName },
               userstate,
             );
+          } else if (msgid === 'raid') {
+            const raider =
+              message.tags['msg-param-displayName'] ||
+              message.tags['msg-param-login'];
+            const viewers = Number(message.tags['msg-param-viewerCount']);
```

Comment:
I propose to change this line to ` const viewers = parseInt(message.tags['msg-param-viewerCount'], 10);`

Number() will return NaN more often & could return floating point numbers.

Reply:
👍

Reply:
I'll make sure to change that, thanks for the heads up!